src/pudl/transform/ferc714.py

```diff
@@ -321,7 +321,7 @@ def _post_process(df: pd.DataFrame, table_name: str) -> pd.DataFrame:
     Returns:
         The post-processed dataframe.
     """
-    return Package.from_resource_ids().get_resource(table_name).enforce_schema(df)
+    return PUDL_PACKAGE.get_resource(table_name).enforce_schema(df)
```

Comment:
Could add an encoding step here and have it apply to all FERC-714 tables. I don't think there are currently any encoded values though.